patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -165,7 +165,7 @@
 +                items.add(itemstack);
              }
 +
-+            net.minecraftforge.event.ForgeEventFactory.fireBlockHarvesting(items, p_180657_1_, p_180657_3_, p_180657_1_.func_180495_p(p_180657_3_), 0, 1.0f, true, p_180657_2_);
++            net.minecraftforge.event.ForgeEventFactory.fireBlockHarvesting(items, p_180657_1_, p_180657_3_, p_180639_3_, 0, 1.0f, true, p_180657_2_);
 +            for (ItemStack stack : items)
```

Comment:
Just edited the file in place. From your reaction, I assume this is not correct. I was tossing up whether to try and describe the problem using text, or show the problem using a pull request.

I now realise that this probably was not the wisest course of action.

Should I make a minimal mod to show the problem?